wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,723 @@
+
+*Notice to translators: English articles with this tag should be avoided. This tag applies when the translation needs its own clean up and must not be blindly copied over from the English article.*
+
+For example:
+
+```yaml
+needs_cleanup: true
+```
+
+When marking articles with this, [comments](#comments) should accompany this to explain what needs to be cleaned.
+
+### Outdated articles
+
+*Notice to translators: Avoid translating English articles with this tag.*
+
+Translated articles that are outdated must use the `outdated` mark when the English variant is updated. English articles may also become outdated when the content they contain is misleading or no longer relevant. For example:
+
+```yaml
+outdated: true
+```
+
+*Note: If the English article has this tag, all translations must have this tag as well.*
+
+When marking articles with this, [comments](#comments) should accompany this to explain what content is outdated.
+
+### Tagging articles
+
+Tags help the website's search engine query articles better. Tags should be written in the same language as the article and include the English's list of tags, and prefer lowercase letters where applicable.
+
+For example, an article called "Skinning tutorial" might include these tags:
+
+```yaml
+tags:
+  - guide
+  - how to
+  - skins
+```
+
+### Translations without reviews
+
+Sometimes, translations are added to the wiki without review from other native speakers of the language. In this case, the `no_native_review` mark is added to let future translators know that it may need to be checked again. For example:
+
+```yaml
+no_native_review: true
+```
+
+*Note to translators: Wiki maintainers will determine and apply this mark prior to merging.*
+
+## Article naming
+
+*See also: [Folder names](#folder-names) and [Titles](#titles)*
+
+Article titles should be singular. See [Wikipedia's naming conventions article](https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(plurals) "Wikipedia") for more details.
+
+Article titles should match the folder name it is in (spaces may replace underscores (`_`) where appropriate). If the folder name changes, the article title should be changed to match it and vise versa. Some exceptions include:
+
+- Contest articles - must use abbreviations, acronyms, or initialisms
+- Tournament articles - must use abbreviations, acronyms, or initialisms
+
+## Folder and file structure
+
+### Folder names
+
+*See also: [Article naming](#article-naming)*
+
+Folder names must be in English and use sentence case. They should match their English article titles as well.
+
+Folder names must only use these characters:
+
+- uppercase and lowercase letters
+- numbers
+- underscores (`_`)
+- exclamation marks (`!`)
+
+#### Percent encoding characters
+
+Folder names must not use characters that require percent encoding. Those characters may include, but are not limited to:
+
+- spaces
+  - all spaces must be changed to underscores (`_`)
+- apostrophes (`'`)
```

Comment:
```suggestion
- spaces
  - all spaces must be changed to underscores (`_`)
- apostrophes (`'`)
- hyphens
```

Reply:
Looking at this section again, I am going to remove this section instead because the higher section above already lists the characters that are acceptable. However, I do think the hyphen is fine to use (although it probably wouldn't be used often at all).